src/main/scala/abalone/Pos.scala

```diff
@@ -310,10 +310,11 @@ object Pos {
     case _                  => (_:Pos) => None
   }
 
-  def sideMovesDirsFromDir(dir: Direction): Directions = {
+  // used by valid moves generator, based on the lineDir currently considered
+  def sideMoveDirsFromLineDir(lineDir: Direction): Directions = {
```

Comment:
This function looks a bit suspicious to me. I would at least like a comment as to why E5 is hardcoded into this. Is it because its the centre of the board?
Is it definitely safe to `getOrElse` to `A1`? Does that mean we always end up in line 324?